extract_unique_sensors_01.py

```diff
@@ -48,7 +56,8 @@ def read_sensors(file_name, url):
             sensors_url = urlopen(url)
```

Comment:
You could also log when filename or url is used...
Perhaps, `debug` level.

Reply:
Done, logged when url and filename are being used with the debug log level.